src/toil/wdl/wdltoil.py

```diff
@@ -92,7 +94,13 @@ def wdl_error_reporter(task: str, exit: bool = False, log: Callable[[str], None]
         WDL.Error.ImportError,
         WDL.Error.ValidationError,
         WDL.Error.MultipleValidationErrors,
-        FileNotFoundError
+        FileNotFoundError,
+        # Don't expose tracebacks to the user for exceptions that may be expected
```

Comment:
This comment applies to all these exceptions, really. I might put it inside the except block; BME205 taught that inside the block is the best place for comments describing the whole control structure.